yum.conf.in

```diff
@@ -1,6 +1,6 @@
 [main]
 cachedir=/var/cache/yum/clear/
-keepcache=1
+keepcache=0
```

Comment:
I would like a longer commit message to explain what these "interesting errors" actually are.

Reply:
If I run into the issue again I will update the commit message. It is something similar to this: https://bugzilla.redhat.com/show_bug.cgi?id=877332 (although not exactly the same bug, the traceback isn't exactly the same).

Clearing the cache and running again seems to fix the issue.